src/settings/PatronNotices/PatronNoticeForm.js

```diff
@@ -81,9 +81,11 @@ class PatronNoticeForm extends React.Component {
 
     if (name) {
       try {
-        const response = await this.getTemplatesByName(name);
+        const response = await this.getTemplatesByName(name.toLowerCase());
```

Comment:
Here we can to not format string `toLowerCase` since the same is happening below, right?

Reply:
The fetch in `this.getTemplatesByName()` is case insensitive (try it yourself for the query in question, `templates?query=(name=="${name}")`**) so `.toLowerCase()` is irrelevant here. It's only the comparison in the `find()` function below that matters, and that looks correct to me. 

** When a field gets a full-text index, an [exact-match search with `==` ignores case](https://github.com/folio-org/raml-module-builder#cql-relations). I can see the logic behind this (e.g. typing in "harry potter") but it surprised me at first because I was thinking of template-names like hash-keys somehow, but really they're not.